cmd/opentelemetry-collector/main.go

```diff
@@ -104,6 +107,7 @@ func main() {
 // getOTELConfigFile returns name of OTEL config file.
 func getOTELConfigFile() string {
 	f := &flag.FlagSet{}
+	f.SetOutput(ioutil.Discard)
```

Comment:
What is the reason for adding this?

Reply:
This can print out an error in some situations. `getOTELConfigFile` is a hack that parses flags before actual parsing by cobra command so it's fine to ignore it. If there is an error it is recognized once the command runs.